cg/store/filters/status_case_filters.py

```diff
@@ -126,9 +126,37 @@ def filter_cases_by_customer_entry_id(cases: Query, customer_entry_id: int, **kw
     return cases.filter(Customer.id == customer_entry_id)
 
 
+def filter_cases_by_customer_entry_ids(
+    cases: Query, customer_entry_ids: List[int], **kwargs
+) -> Query:
+    """Return cases with matching customer ids."""
+    return cases.filter(Customer.id.in_(customer_entry_ids)) if customer_entry_ids else cases
+
+
+def filter_cases_by_action(cases: Query, action: str, **kwargs) -> Query:
+    """Return cases with matching action."""
+    return cases.filter(Family.action == action) if action else cases
```

Comment:
wont this statement always return the initial query i.e. all cases when there is no action? Is this wanted behaviour?

Reply:
This makes chaining filters easier since we have filter functions which take in many optional parameters. Instead of checking whether each argument is passed we can just define a list of all filters.

Reply:
I'll keep it as it is, but we should be wary of any instances where want to filter by None.